src/js/node/http.ts

```diff
@@ -1209,6 +1209,15 @@ class ClientRequest extends OutgoingMessage {
   #protocol;
   #method;
   #port;
+  #pfx = null;
```

Comment:
Instead of top-level keys that exist on every object, lets move this to a separate object since it is less frequently allocated and only mark that one as null (or undefined)

Reply:
https://github.com/oven-sh/bun/pull/6621/commits/b8d66e7edc45eee4bb606ed343dd93fadaae0828#diff-91e6e16a6a18c42ea259bb91cd801b3707bbbf4f8c8acb48cc3df2152c02d1bd